packages/lesswrong/components/posts/PostsItemComments.tsx

```diff
@@ -22,12 +25,14 @@ const styles = (theme: ThemeType): JssStyles => ({
     position: "relative",
     flexShrink: 0,
     top: 2,
+    '& div': {
+      marginTop: -3,
+    }
   },
   commentCount: {
     position:"absolute",
     right:"50%",
     top:"50%",
-    marginTop:-3,
```

Comment:
I reworked this margin to be dependent on the comment icon size, since it seemed too high in the smaller case:

![image](https://user-images.githubusercontent.com/25752873/123343273-4dfedb00-d506-11eb-927e-68fb2ae48276.png)